scss/boost_union/post.scss

```diff
@@ -194,6 +231,21 @@
     #searchform-navbar {
         background-color: $primary;
     }
+
+    /* Change the color of the login link in the navbar to dark grey and to darker grey when hovered. */
+    .login,
+    .login a {
+        color: rgba($black, .6);
+        &:hover {
+            color: rgba($black, .9);
+        }
+    }
+
```

Comment:
Try to prepend this by

/*csslint important:false*/

I think you give the rationale to use this in the comment. The above line would then shut csslint quiet.
Here's a usage example: https://github.com/moodle/moodle/blob/MOODLE_402_STABLE/lib/yuilib/3.18.1/cssnormalize/cssnormalize.css#L565

Thanks for your contribution, @wa-bea !

Best,
Luca

Reply:
Please see my comment above :)